src/listing.rs

```diff
@@ -169,25 +169,10 @@ pub fn directory_listing(
     dirs_first: bool,
     hide_version_footer: bool,
     title: Option<String>,
-) -> Result<ServiceResponse, io::Error> {
+) -> io::Result<ServiceResponse> {
     use actix_web::dev::BodyEncoding;
     let serve_path = req.path();
 
-    // In case the current path is a directory, we want to make sure that the current URL ends
-    // on a slash ("/").
```

Comment:
Is this now handled directly by actix-web?

Reply:
yes, see the last commit. It now uses `Files::redirect_to_slash_directory()`